packages/sdk/src/reads/getDelegates.ts

```diff
@@ -0,0 +1,60 @@
+import { AAVE, COMP, STKAAVE, UNI } from "../../tests/constants.js";
+import { type ReadContractParameters, readContractParameters } from "../utils/viem.js";
+import { type Address, type PublicClient, parseAbi } from "viem";
+
+export async function getDelegates(
+  client: PublicClient,
+  args: ReadContractParameters<{
+    vaultProxy: Address;
+  }>,
+) {
+  const tokenDaoDelegates = parseAbi(["function delegates(address account) view returns (address)"] as const);
+
+  const tokenDaoDelegateeByType = parseAbi([
+    "function getDelegateeByType(address delegator, uint8 type) view returns (address)",
+  ] as const);
+
+  const contractsWithoutVotingPower = [{ address: COMP }, { address: UNI }];
```

Comment:
I would just do `const contractsWithoutVotingPower = [COMP, UNI ] as const;`, there is no benefit from nesting